.github/workflows/validate.yml

```diff
@@ -10,22 +10,28 @@ jobs:
   commitlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout The Repository
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: wagoid/commitlint-github-action@v4
+      - name: Commit linting
+        uses: wagoid/commitlint-github-action@v4
 
   validate:
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js version 16
+      - name: Checkout The Repository
+        uses: actions/checkout@v3
+
+      - name: Use Node.js version 16.x
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: '16.x'
           cache: 'yarn'
-      - run: yarn install
-      - run: yarn deduplicate:check
-      - run: yarn prepublishOnly
-      - run: yarn type-check
+
+      - name: Install Dependencies & Build & Validate
+        run: |
```

Comment:
why did you merged these runs together ?

Reply:
What do you mean merge?
That's a shorter syntax to run sequential ops.